04-vector-raster-integration-advanced.Rmd

```diff
@@ -159,29 +156,37 @@ dev.off()
 ```
 
 The graphic above illustrates the extent of several of the spatial layers that 
-we have worked with in this vector data lesson series including:
+we have worked with in this vector data lesson series:
 
-* Area of interest (AOI).
-* A roads/trails.
-* And plot locations.
-* A canopy height model (CHM) in GeoTIFF format.
+* Area of interest (AOI) -- blue
+* Roads and trails -- purple
+* Vegetation plot locations -- black
 
-However, sometimes we have a raster file that is must larger than our study 
-area or area of interest. It is often more efficient to `crop` the raster to the 
-extent of our study area, to keep file sizes smaller as we process our data. 
+and a raster file, that we will introduce this lesson: 
 
+* A canopy height model (CHM) in GeoTIFF format -- green
 
-## Import Data
-We will use four vector shapefiles with data on (field site boundary,
-roads/trails, tower location, and study plot locations) and one raster GeoTIFF
-file, a Canopy Height Model for the Harvard Forest, Massachusetts. 
+The raster files that we have to work with will rarely match the desired vector 
+extent (our study area or area of interest) perfectly.  We often need to `crop` 
+the raster to the extent of our study area to keep file sizes smaller as we
+process our data. 
+
+### Import Data
+We will use four vector shapefiles (field site boundary, roads/trails,
+tower location, and veg study plot locations) and one raster
+GeoTIFF file, a Canopy Height Model for the Harvard Forest, Massachusetts, to 
+learn how to crop rasters using the vector extent. 
```

Comment:
that text doesn't make sense because it still suggests they are using 4 files to crop a raster. 

Please replace with: 

We will begin by importing four vector shapefiles (field site boundary, roads/trails, tower location, and veg study plot locations) and one raster GeoTIFF file, a Canopy Height Model for the Harvard Forest, Massachusetts. These data can be used to create maps that characterize our study location.

<<the above text is more general - they are just opening the data in the first chunk>